src/Utils/Hooks/useAuthIntent/mutations/AuthIntentAssociateSubmissionMutation.ts

```diff
@@ -0,0 +1,33 @@
+import { commitMutation, Environment, graphql } from "relay-runtime"
+import { AuthIntentAssociateSubmissionMutation } from "__generated__/AuthIntentAssociateSubmissionMutation.graphql"
+import { AuthIntentMutation } from "./types"
+
+export const associateSubmissionMutation: AuthIntentMutation = (
```

Comment:
this might just be a refactor, but why not use our `useMutation` hook?

Reply:
Yeah I left a TODO about it; just didn't feel like going to crazy here. The specs, etc.

Reply:
yah np at all